extension.js

```diff
@@ -86,43 +87,38 @@ function onScroll(actor, event) {
   return Clutter.EVENT_STOP;
 }
 
-
 function createOverlay() {
   if(overlayExists) return;
   Log.debug("overlay created");
-  overlayContainer = new St.Widget({
-    clip_to_allocation: true,
-    layout_manager: new Clutter.BinLayout()
-  });
-  overlayContainer.add_constraint(new Layout.MonitorConstraint({primary: true, work_area: true}));
```

Comment:
I think overlayContainer is not needed anymore because it applies to all monitors now?

Reply:
Yes. I assumed that you used overlayContainer so that the overlay is created on the primary monitor. I think it is not needed now. If there was another reason for overlayContainer and the absence of it will create problems, I can look into that.